calculator.js

```diff
@@ -1,13 +1,72 @@
+console.log('Welcome!')
 
-const exampleAdditionInput = {
+const sampleUserInput = {
   num1: 3,
-  num2: 5,
-  operation: 'add',
+  num2: 0,
+  operation: 'divide',
+}
+
+const validateInput = function(input) {
+  if (isNaN(input.num1)) {
+    console.log(`${input.num1} is not a number`);
+  }
+  else if (isNaN(input.num2)) {
+    console.log(`${input.num2} is not a number`);
+  }
+  else {
+    return true;
+  }
 }
 
 const calculateUserInput = function (error, promptInput) {
-  console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
+  // console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
+
+  const printInput = function(input, operation, result) {
+    console.log(`${input.num1} ${operation} ${input.num2} = ${result}`);
+  }
+
+  const addNums = function(num1, num2) {
+    return parseInt(num1) + parseInt(num2);
+  }
 
+  const subtractNums = function(num1, num2) {
+    return parseInt(num1) - parseInt(num2);
+  }
+
+  const multiplyNums = function(num1, num2) {
+    return parseInt(num1) * parseInt(num2);
+  }
+  
+  const divideNums = function(num1, num2) {
+    return (parseInt(num2) === parseInt(0) ? console.log('You cannot divide by zero') : parseInt(num1) / parseInt(num2));
+  }
+    
+  switch(promptInput.operation) {
+    case '+':
+      case 'add':
+      result = addNums(promptInput.num1, promptInput.num2);
+      printInput(promptInput, '+', result)
+      break;
+    case '-':
+      case 'subtract':
+      result = subtractNums(promptInput.num1, promptInput.num2);
+      printInput(promptInput, '-', result)
+      break;
+    case '*':
```

Comment:
great use of a switch statement, love the use of overflows too! Just a note, we keep all `case x:` sections indented the same, like so: 
```suggestion
    case '+':
    case 'add':
      result = addNums(promptInput.num1, promptInput.num2);
      printInput(promptInput, '+', result)
      break;
    case '-':
    case 'subtract':
      result = subtractNums(promptInput.num1, promptInput.num2);
      printInput(promptInput, '-', result)
      break;
    case '*':
```